CHANGELOG.md

```diff
@@ -1,13 +1,3 @@
-<a name="1.15.6"></a>
```

Comment:
Is this change intended?

Reply:
@lord2800 yes, this is part or "rolling back" from the release that we made and re-release 1.15.6

Reply:
Got it.